pl.json

```diff
@@ -1653,128 +1653,128 @@
-			"action-edit-background": "Edit background",
-			"action-replace-background": "Replace background",
-			"action-remove-background": "Remove background",
-			"label-no-images": "No images found.",
+			"action-arrange-grid": "Ułóż w siatce",
+			"action-export-png": "Wyeksportuj jako obraz",
+			"action-jump-to-group": "Przeskocz do grupy",
+			"action-set-background": "Ustaw tło",
+			"action-edit-background": "Edytuj tło",
+			"action-replace-background": "Zmień tło",
+			"action-remove-background": "Usuń tło",
+			"label-no-images": "Nie znaleziono obrazów.",
 			"label-export-png-desc": "Export \"{{title}}\" with a PNG file with the settings below.",
-			"label-export-png-dimensions": "Estimated image dimensions: {{dimensions}}",
-			"label-distribute": "Distribute",
-			"label-justify": "Justify",
-			"label-arrange": "Arrange",
-			"label-always": "Always",
-			"label-hover": "On hover",
-			"label-never": "Never",
-			"label-no-section-found": "No section found",
-			"option-wheel-behavior": "Default mouse wheel behavior",
-			"option-mod-drag-behavior": "Default {{key}} behavior",
-			"option-show-menu": "Show menu",
-			"option-new-canvas-location": "Default location for new canvas files",
-			"option-new-canvas-location-description": "Where newly created canvas files are placed.",
-			"option-new-canvas-folder-path": "Folder to create new canvas files in",
-			"option-new-canvas-folder-path-description": "Newly created canvas files will appear under this folder.",
-			"option-node-label": "Display card label",
-			"option-snap-to-grid": "Snap to grid",
-			"option-snap-to-grid-desc": "Snap cards to the background grid when moving and resizing.",
-			"option-snap-to-objects": "Snap to objects",
-			"option-snap-to-objects-desc": "Snap cards to nearby objects when moving and resizing.",
-			"option-zoom-breakpoint": "Zoom threshold for hiding card content",
-			"option-zoom-breakpoint-desc": "Lower values will increase performance but hide card content sooner when zooming out.",
-			"option-export-png-show-logo": "Show logo",
-			"option-export-png-show-logo-desc": "This will add an Obsidian logo to the bottom left.",
-			"option-export-png-privacy-mode": "Privacy mode",
-			"option-export-png-privacy-mode-desc": "This will obscure any text on your canvas.",
-			"option-export-png-zoom": "Zoom",
-			"option-export-png-zoom-desc": "A higher zoom will generate a higher resolution image.",
-			"option-export-png-frame": "Viewport",
-			"option-export-png-frame-desc": "Choose to render the entire canvas or just the current visible viewport.",
-			"option-export-png-frame-full": "Full canvas",
-			"option-export-png-frame-viewport": "Viewport only",
+			"label-export-png-dimensions": "Szacowane rozmiary obrazka: {{dimensions}}",
+			"label-distribute": "Rozłóż",
+			"label-justify": "Wyjustuj",
+			"label-arrange": "Ułóż",
+			"label-always": "Zawsze",
+			"label-hover": "Po najechaniu kursorem",
+			"label-never": "Nigdy
```

Comment:
Could you please fix the JSON formatting error here?

Reply:
Typo is fixed, sorry for that :)